src/middlewared/middlewared/schema/attribute.py

```diff
@@ -1,8 +1,10 @@
 import copy
 import json
 import textwrap
+import typing
```

Comment:
To be sure, we are not right now completing type hints here and would be done at a later stage ?